Sundar2021reproducibility

varun19299 opened the pull request:
Hi, we got done till step 4, but with a few issues:

* author names don't show up in the [PDF](https://github.com/ReScience/RC-2020/blob/bb647fa78228c2c89c46f4e53559a1fbaeaa3b4d/sundar2021reproducibility/journal/article.pdf)
* Citations, links aren't coloured (should we remove xcolor?)

koustuvsinha commented:
Hi @varun19299 , yes please remove `xcolor` :)

koustuvsinha commented:
@varun19299 can you paste your build.log? I will inspect the author name issue

varun19299 commented:
Would this be [author.log](https://github.com/varun19299/RC-2020/blob/main/sundar2021reproducibility/journal/article.log) or is there another build.log file?

koustuvsinha commented:
Hi Varun,

It would be the output of make command. To capture it, run “make > build.log”.

Thanks,

Koustuv

On Apr 9, 2021, at 11:42 PM, Varun Sundar ***@***.***> wrote:

﻿

Would this be author.log<https://github.com/varun19299/RC-2020/blob/main/sundar2021reproducibility/journal/article.log> or is there another build.log file?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/ReScience/RC-2020/pull/2#issuecomment-817071804>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABHCWBUPMSPWQTOB6VBEQI3TH7CL3ANCNFSM42VFDQBQ>.

varun19299 commented:
Fixed author names by setting abstract as empty string in `metadata.yaml`

varun19299 commented:
Also resized a couple of tables. We might reshape another figure to better suit the layout.

varun19299 commented:
Here's the [build.log](https://gist.github.com/varun19299/5da1e8c8d4159c408454c43ca8b58382).

koustuvsinha commented on sundar2021reproducibility/openreview/packages.tex:
```diff
@@ -0,0 +1,70 @@
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{graphicx}      % images
+\usepackage[english]{babel}
```
It seems this import is the issue from the logs: "! LaTeX Error: Option clash for package babel." Can you disable this and re-build?

koustuvsinha commented on sundar2021reproducibility/openreview/packages.tex:
```diff
@@ -0,0 +1,70 @@
+% \usepackage[nonatbib, final]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Please remove this import `fontenc` as this appears to break ReScience fonts (check #4 for discussions)

koustuvsinha commented:
> Can citation links be coloured?

Unfortunately, it seems rescience.cls does not support colored citation links.

varun19299 commented:
Removed these packages. Updated [PDF](https://github.com/ReScience/RC-2020/blob/edfa0a92b2f260977e27941c725dbe67f2d0808b/sundar2021reproducibility/journal/article.pdf).

Any suggestions for fixing the inline math overflow on page 9? (\alpha, \Delta T =...)

koustuvsinha commented on sundar2021reproducibility/openreview/packages.tex:
```diff
@@ -0,0 +1,76 @@
+% Disable for Rescience C
+% \usepackage[english]{babel}
+
+\bibliographystyle{plainnat}
```
Disable this import, seems to break compilation on my end

varun19299 replied:
Done in [commit](https://github.com/ReScience/RC-2020/pull/2/commits/d8f404ed4db5a7deccdb844d0d6ff6369b1f6255).

koustuvsinha commented on sundar2021reproducibility/openreview/sections/methodology.tex:
```diff
@@ -0,0 +1,29 @@
+The authors provide publicly accessible code\footnote{\href{https://github.com/google-research/rigl}{https://github.com/google-research/rigl}} written in Tensorflow (\citet{abadi2016tensorflow}). To gain a better understanding of various implementation aspects, we opt to replicate \textit{RigL} in Pytorch (\citet{Pytorch}). Our implementation extends the open-source code\footnote{\href{https://gi ...
+
+
+\subsubsection{Mask Initialization}}For a network with $L$ layers and total parameters $N$, we associate each layer with a random boolean mask of sparsity $s_l, \; l \in [L]$. The overall sparsity of the network is given by $S=\frac{\sum_l s_l N_l}{N}$, where $N_l$ is the parameter count of layer $l$. Sparsities $s_l$ are determined by the one of the following mask initialization strategies:
```
There is an extra `}` which is causing latex overflow of equation in page 9

varun19299 replied:
Fixed this in [commit](https://github.com/ReScience/RC-2020/pull/2/commits/d8f404ed4db5a7deccdb844d0d6ff6369b1f6255).

koustuvsinha commented:
Thanks for the changes! LGTM

varun19299 commented:
Hi @koustuvsinha, we're done making all changes from our end. Please let us know if anything else is pending.

koustuvsinha commented:
LGTM on the latex side. @dodgejesse can you review the content side? Then we will be good to go.

dodgejesse reviewed (approved):
Looks good to me. The authors could optionally remove the whitespace from the page break on the second page (after the summary, before the intro), but that's just an aesthetic choice. It looks like an appendix has also been added, which is fine. Good work!
